features/bevragen/persoon/overlijden/in-onderzoek.feature

```diff
@@ -0,0 +1,106 @@
+    | inOnderzoek.land                             | <land io>   |
+    | inOnderzoek.datumIngangOnderzoek.type        | Datum       |
+    | inOnderzoek.datumIngangOnderzoek.datum       | 2002-07-01  |
+    | inOnderzoek.datumIngangOnderzoek.langFormaat | 1 juli 2002 |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | datum io | plaats io | land io | type                      |
+    | 060000                  | true     | true      | true    | hele categorie overlijden |
+    | 060800                  | true     | true      | true    | hele groep overlijden     |
+    | 060810                  | true     |           |         | datum overlijden          |
+    | 060820                  |          | true      |         | plaats overlijden         |
+    | 060830                  |          |           | true    | land overlijden           |
+
+  Abstract Scenario: '<type>' is in onderzoek en '<field>' veld wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'overlijden' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    | datum overlijden (08.10)        | 20020701                  |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | overlijden.<field>              |
+    Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+    | naam                                         | waarde      |
+    | datum.type                                   | Datum       |
+    | datum.datum                                  | 2002-07-01  |
+    | datum.langFormaat                            | 1 juli 2002 |
+    | inOnderzoek.<field>                          | true        |
+    | inOnderzoek.datumIngangOnderzoek.type        | Datum       |
+    | inOnderzoek.datumIngangOnderzoek.datum       | 2002-07-01  |
+    | inOnderzoek.datumIngangOnderzoek.langFormaat | 1 juli 2002 |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | field | type                      |
+    | 060000                  | datum | hele categorie overlijden |
+    | 060800                  | datum | hele groep overlijden     |
+    | 060810                  | datum | datum overlijden          |
+
+  Abstract Scenario: '<type>' is in onderzoek en '<field>' veld wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'overlijden' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    | datum overlijden (08.10)        | 20020701                  |
```

Comment:
vanuit testperspectief is het een beetje jammer dat deze twee datums gelijk zijn

Reply:
aangepast